lib/ansible/module_utils/urls.py

```diff
@@ -81,11 +81,18 @@
 # agrees to be bound by the terms and conditions of this License
 # Agreement.
 
+# FIXME: try to use six.moves once we exclusively use ziploader
+HTTPSHandler = None
 try:
-    import urllib2
+    from urllib2 import urlopen, Request, HTTPError, URLError, BaseHandler, HTTPPasswordMgrWithDefaultRealm, HTTPSHandler, AbstractHTTPHandler, HTTPRedirectHandler, build_opener as url_build_opener, install_opener as url_install_opener, HTTPBasicAuthHandler, OpenerDirector
```

Comment:
Would it be preferable to split the import over multiple lines for readability ? ie. 

```
            from urllib2 import (
    urlopen, 
    Request, 
    HTTPError, 
    URLError, 
    BaseHandler, 
    HTTPPasswordMgrWithDefaultRealm, 
    HTTPSHandler, 
    AbstractHTTPHandler, 
    HTTPRedirectHandler, 
    build_opener as url_build_opener, 
    install_opener as url_install_opener, 
    HTTPBasicAuthHandler, 
    OpenerDirector,
)
```